test/mailers/previews/comment_mailer_preview.rb

```diff
@@ -165,7 +116,111 @@ def admin_edited_comment_notification_unreviewed
     admin = create(:admin)
 
     commentable = create(:admin_post, moderated_commenting_enabled: true)
-    comment = create(:comment, commentable: commentable, unreviewed: true)
+    comment = create(:comment, commentable: commentable, edited_at: Time.current, unreviewed: true)
     CommentMailer.edited_comment_notification(admin, comment)
   end
+
+  # Sent to a user when they get a comment on a titled chapter of a chaptered work
+  def chapter_comment_notification_titled
```

Comment:
Do you think it would be doable to have these commentable types be done similar to the different types for the creatorship notifications in the user_mailer_preview, so defining an array of types, factories for those types, and looping over that to define the methods?

Also, I think we messed up here and should have put the commentable type as a suffix....